.github/CODEOWNERS

```diff
@@ -9,15 +9,8 @@
 # Resources
 #  - [CODEOWNERS] https://help.github.com/articles/about-codeowners
 
-*       @rajaguruduraisamy-okta @ganeshsomasundaram-okta @jmelberg-okta @jeffbelser-okta @santhoshbalakrishnan 
+*               @okta/design-system
 
-*.md    @okta/design-system
-*.scss  @okta/design-system
-*.html  @okta/design-system
-*.swig  @okta/design-system
-*.woff  @okta/design-system
-*.woff2 @okta/design-system
-*.svg   @okta/design-system
-*.css   @okta/design-system
-*.png   @okta/design-system
-*.gif   @okta/design-system
+package.json    @rajaguruduraisamy-okta @ganeshsomasundaram-okta @jmelberg-okta @jeffbelser-okta @santhoshbalakrishnan @nikhilvenkatraman-okta
```

Comment:
Let's also add `.github/workflows/` and `scripts/` to this list. That way any changes to artifact publishing are reviewed by the UI Core team.